cmd/run/run.go

```diff
@@ -508,7 +253,7 @@ func ReadConfig() (*Config, error) {
 	return config, nil
 }
 
-func VerifyConfig(cfg *Config) error {
+func VerifyConfig(cfg *serverconfig.Config) error {
 	if int(cfg.MaxConcurrentReadsForCheck) > cfg.Datastore.MaxOpenConns {
```

Comment:
Please drop this and the one below it for the ListObjects max concurrent reads. They are currently emitted every time you start the server because the defaults changed without changing this somehow 🤷 

The default is unbounded/unthrottled (math.MaxUint32) reads per Check and ListObjects requests, which will obviously be higher than the default max open conns. So let's drop this.